source/partials/primary-domain.md

```diff
@@ -1,8 +1,19 @@
+### Set a Primary Domain via the Dashboard
+
+1. From the environment you want to set a primary domain for (typically Test or Live), navigate to **<span class="glyphicons glyphicons-global"></span> Domains / HTTPS**.
+
+1. Ensure that all domains have been added and are listed.
+
+1. In the **Choose Primary Domain** section, select the domain to which traffic should be redirected, and click **Save Configuration**.
+
+   - You can track the task in **Workflows** <Icon icon="chevron-down" />.
```

Comment:
Is this line about the workflow needed? Seems like too much info.

Reply:
I'm fine with removing it... @EdwardAngert?

Reply:
+1
tracking the status in Workflows is a Platform given. I can remove it

Reply:
resolved in #5210